website/docs/guide-chapter-instructions.rst

```diff
@@ -464,6 +464,27 @@ When there are only two alleles per locus, :math:`W_n` is equivalent
 to the correlation coefficient between the two loci, defined as
 :math:`r =\sqrt {D_{11} / p_1 p_2 q_1 q_2 }`.
 
+:math:`W_{A/B}` and `W_{B/A}`
+ 
+When there are different numbers of alleles at the two loci,
+the direct correlation property for the :math:`r` correlation 
+measure is not retained by :math:`W_n`, its multi-allelic extension. 
+The complementary pair of conditional asymmetric LD (ALD) measures, 
+`W_{A/B}` and `W_{B/A}`, were developed to extend the `W_n` measure 
+(Thomson and Single, 2014). `W_{A/B}` is (inversely) related to the 
+degree of variaation of A locus alleles on haplotypes conditioned 
+on B locus alleles. If there is no variation of A locus alleles 
+on haplotypes conditioned on B locus alleles, then `W_{A/B} = 1`
+`W_{A/B} = W_{B/A} = `W_n` when there is symmetry in the data and 
+thus for bi-allelic SNPs.
```

Comment:
```suggestion
   When there are different numbers of alleles at the two loci,
   the direct correlation property for the :math:`r` correlation 
   measure is not retained by :math:`W_n`, its multi-allelic extension. 
   The complementary pair of conditional asymmetric LD (ALD) measures, 
   :math:`W_{A/B}` and :math:`W_{B/A}`, were developed to extend the :math:`W_n` measure 
   (Thomson and Single, 2014). :math:`W_{A/B}` is (inversely) related to the 
   degree of variation of A locus alleles on haplotypes conditioned 
   on B locus alleles. If there is no variation of A locus alleles 
   on haplotypes conditioned on B locus alleles, then :math:`W_{A/B} = 1`
   :math:`W_{A/B} = W_{B/A} = W_n` when there is symmetry in the data and 
   thus for bi-allelic SNPs.
```

indenting to match the other blocks, and also add in missing `:math:` directives

Reply:
I think I got everything. I added some updated output with ALD stats. I also added a reference to teh ALD paper and an entry in the biblio.rst

Reply:
close - there are a couple of other minor syntactical fixes related to math that are causing the current build to fail - see my current review - if you commit the changes it should build.  

also better to keep all changes in one pull-request. if they go beyond a single file, then I recommend checking out the repo locally, making changes across multiple files on your computer, and then pushing back to your branch (which will update the pull request).  it's not easy to edit multiple files via the github web UI, unfortunately.